arkouda/pdarraycreation.py

```diff
@@ -34,6 +34,7 @@
     "zeros",
     "ones",
     "full",
+    "_full_string",
```

Comment:
If we are exposing full_string here, I don't know that we need the `_` at the beginning. That is used to denote private methods that are not user facing, but adding it here makes it user facing.

Alternatively, looking at the usage of the method, I think having it as private would be the right call. Based on that we would not need to expose it within the `__all__`

Reply:
Removed